pkg/disttask/framework/taskexecutor/slot.go

```diff
@@ -23,29 +24,22 @@ import (
 // slotManager is used to manage the slots of the executor.
 type slotManager struct {
 	sync.RWMutex
-	// taskID -> slotInfo
-	executorSlotInfos map[int64]*slotInfo
+	executorSlotInfos map[int64]*proto.Task
 
 	// The number of slots that can be used by the executor.
 	// It is always equal to CPU cores of the instance.
 	available int
-}
 
-type slotInfo struct {
-	taskID int
-	// priority will be used in future
-	priority  int
-	slotCount int
+	taskWaitAlloc *proto.Task
```

Comment:
slot manager shouldn't manage which task is waiting, and not needed

Reply:
Do you mean manage it in `tastexecutor.Manager`?

Reply:
> ~Do you mean manage it in `tastexecutor.Manager`?~

No need to manage task since we fetch runnableTask in loop